### DISCUSSION TITLE
4k monitor title bar text is too small

### DISCUSSION THREAD

**Discussion by okaytony** [sumatrapdfreader/sumatrapdf#4724] [General]
I use both 4k and 1080p monitors. When I drag a window from the 1080p monitor to the 4k monitor, the text in the title bar becomes very small. Is there any way to solve this problem?

**Comment by HansBoikon** [comment]
In my case the tabs of the documents go over the window titlebar. This makes it hard to select the titlebar to drag the window to another monitor. What works for me is setting these advanced options:
UseTabs = false

*optional
RememberOpenedFiles = false
FullPathInTitle = true

**Comment by EdMueller** [comment]
I have the same problem. Look at this (left is 4k monitor, right is FullHD monitor):

![20250104_161505](https://github.com/user-attachments/assets/ce376350-e84d-4696-ae71-1bdc4b2fd9a8)

Other software (e.g. the Windows Explorer) resizes the window, so you can read the text ;-).
